apps/stable_diffusion/scripts/outpaint.py

```diff
@@ -200,14 +200,16 @@ def outpaint_inf(
     if args.img_path is None:
         print("Flag --img_path is required.")
         exit()
-    if "inpaint" not in args.hf_model_id:
-        print("Please use inpainting model with --hf_model_id.")
-        exit()
 
     dtype = torch.float32 if args.precision == "fp32" else torch.half
     cpu_scheduling = not args.scheduler.startswith("Shark")
     set_init_device_flags()
-    schedulers = get_schedulers(args.hf_model_id)
+    model_id = (
+        args.hf_model_id
+        if "inpaint" in args.hf_model_id
```

Comment:
Should this be outpaint?

Reply:
Outpainting also uses inpainting models.